.travis/install_amd_sdk.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
maybe my knowledge of the travis API is lacking behind a bit, but why do you need this file in `.travis/`?

Reply:
this file installs the AMD OpenCL SDK (atm it seems to work only with OpenCL v1.2), so we can compile and execute clfft (on CPUs).
This script is executed by `.travis.yml` (`bash .travis/install_amd_sdk.sh ${AMDAPPSDK_VERSION}`).